events/models.py

```diff
@@ -20,17 +24,26 @@ class Event(models.Model):
     registration_last_day_cancel = models.DateTimeField(null=True)
     public_registration = models.BooleanField(default=False)
     allowed_groups = models.ManyToManyField(Group, blank=True)
-    send_submission_mail = models.BooleanField(default=False) 
+    send_submission_mail = models.BooleanField(default=False)
     submission_mail_subject = models.TextField(blank=True)
     submission_mail_body = models.TextField(blank=True)
+    image = models.ImageField(
+            upload_to=random_path('events', 'image'), blank=True)
 
     def __str__(self):
         return '%s'%(self.name)
 
+    def save(self, *args, **kwargs):
+        super(Event, self).save(*args, **kwargs)
```

Comment:
Because the image is first saved and then converted and resized. This is a temporary solution and we are looking for a solution to it but for now it is OK.